boards/arm/mimxrt1170_evk/evkmimxrt1170_connect_cm4_cm4side.jlinkscript

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019-2020 NXP
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Not sure why the scancode check isn't failing on this, @nashif?

I'd like to see if we can relicense these files to Apache, but in the meantime let's remove them from this PR and add back later in a follow up PR.

Reply:
Removed this file